runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkJobServerDriver.java

```diff
@@ -130,7 +137,7 @@ private FlinkJobServerDriver(
     this.configuration = configuration;
     this.executor = executor;
     this.jobServerFactory = jobServerFactory;
-    this.artifactServerFactory = jobServerFactory;
+    this.artifactServerFactory = artifactServerFactory;
```

Comment:
Thanks for spotting this.

Reply:
I've merged this fix to master with fb23c7f2eacee5374cfd85a58d5d6746e18fabc7.

Reply:
Thanks!